### ISSUE TITLE
The example project doesn't run on Flutter Master branch

### ISSUE THREAD

**Issue by Pacane** [dukhanov/flutter-nfc-plugin#1]
```
D/FlutterNfcPlugin(26466): callback: onTagDiscovered {error=Cannot parse NDEF message: java.lang.IllegalStateException: ndef must not be null, message=null}
W/Binder  (26466): Caught a RuntimeException from the binder stub implementation.
W/Binder  (26466): java.lang.RuntimeException: Methods marked with @UiThread must be executed on the main thread. Current thread: Binder:26466_3
W/Binder  (26466): 	at io.flutter.embedding.engine.FlutterJNI.ensureRunningOnMainThread(FlutterJNI.java:605)
W/Binder  (26466): 	at io.flutter.embedding.engine.FlutterJNI.dispatchPlatformMessage(FlutterJNI.java:515)
W/Binder  (26466): 	at io.flutter.embedding.engine.dart.DartMessenger.send(DartMessenger.java:76)
W/Binder  (26466): 	at io.flutter.embedding.engine.dart.DartExecutor.send(DartExecutor.java:151)
W/Binder  (26466): 	at io.flutter.view.FlutterNativeView.send(FlutterNativeView.java:144)
W/Binder  (26466): 	at io.flutter.plugin.common.EventChannel$IncomingStreamRequestHandler$EventSinkImplementation.success(EventChannel.java:226)
W/Binder  (26466): 	at it.dukhanov.flutter.nfc_plugin.NfcPlugin.sendNfcListenerCallback(NfcPlugin.kt:92)
W/Binder  (26466): 	at it.dukhanov.flutter.nfc_plugin.NfcPlugin.onTagDiscovered(NfcPlugin.kt:82)
W/Binder  (26466): 	at android.nfc.NfcActivityManager.onTagDiscovered(NfcActivityManager.java:447)
W/Binder  (26466): 	at android.nfc.IAppCallback$Stub.onTransact(IAppCallback.java:126)
W/Binder  (26466): 	at android.os.Binder.execTransactInternal(Binder.java:1021)
W/Binder  (26466): 	at android.os.Binder.execTransact(Binder.java:994)
```

**Comment by dukhanov** [comment]
Hi,

the same issue I guess: https://github.com/g123k/flutter_plugin_device_apps/issues/11

**Comment by dukhanov** [comment]
Fixed in version 0.0.3
